### PR TITLE
add-submission-mode

### PR DESCRIPTION
**Pull request by oanaBiancaSchiopu**

### CODE REVIEW

File: scheduling-api-graphql/scheduling-api-graphql-schema/src/main/java/org/ow2/proactive/scheduling/api/graphql/schema/type/inputs/JobInput.java [hunk 1]
```diff
@@ -121,6 +122,10 @@ public GraphQLInputType buildType(DataFetcher... dataFetchers) {
                                                                .description("Bucket name which the job belongs to.")
                                                                .type(GraphQLString)
                                                                .build())
+                                   .field(newInputObjectField().name(SUBMISSION_MODE.getName())
+                                                               .description("Submission mode of the ob.")
```
Comment (fviale) [inline scheduling-api-graphql/scheduling-api-graphql-schema/src/main/java/org/ow2/proactive/scheduling/api/graphql/schema/type/inputs/JobInput.java:126]: some words are missing in this description

**Review by fviale — APPROVED** [review]
LGTM, one minor comment

**Comment by oanaBiancaSchiopu** [comment]
retest this please

**Comment by oanaBiancaSchiopu** [comment]
retest this please

**Pull request merged**